CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to FingerprintJS Pro Azure Integration
+
+## Working with code
+
+We prefer using [yarn](https://yarnpkg.com/) for installing dependencies and running scripts.
+
+For proposing changes, use the standard [pull request approach](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). It's recommended to discuss fixes or new functionality in the Issues, first.
+
+* The `main` and `develop` branches are locked for the push action.
+* Releases are created from the `main` branch. If you have Azure Integration set up, it is running code from the `main` branch. Pull requests into the `main` branch are not accepted.
+* The `develop` branch functions as a candidate for the next release. Create your pull requests into this branch. The code in `develop` must always pass the tests.
+
+### How to build
+* After cloning the repository, run `yarn install` to install dependencies.
+
+* Run `yarn build` to build the project into the `dist` folder. The created `dist/fingerprintjs-pro-azure-function/fingerprintjs-pro-azure-function.js` and `dist/fingerprintjs-pro-azure-function-management/fingerprintjs-pro-azure-function-management.js` are meant to be deployed to Azure Functions.
+
```

Comment:
Let's also mention how to run the functions locally:

1. Run `yarn watch`
2. Run `yarn emulate-storage` - storage emulation is required to run the functions locally
3. Run `yarn start`